docs/algorithms.rst

```diff
@@ -347,27 +340,231 @@ This speeds up mixing and reduces the total number of samples you need to get un
 Markov chain Monte Carlo
 ========================
 
+YANK operates with additional sampling enhancements based on Markov Chain Monte Carlo (MCMC) techniques, and implements
```

Comment:
Awkward. Maybe:
"YANK uses a replica exchange scheme with Gibbs sampling (CITE) to sample multiple thermodynamic states in a manner that improves mixing of the overall Markov chain.
In between replica exchange attempts, YANK uses compositions of MCMC moves from the ```openmmtools.mcmc`` module <http://openmmtools.readthedocs.io/en/latest/mcmc.html>`_ to sample from the equilibrium distribution at each thermodynamic states."